.github/workflows/build-publish-pr.yml

```diff
@@ -29,7 +49,7 @@ jobs:
           aws-region: ${{ secrets.AWS_REGION }}
           sign-images: false
           ecr-hostname: ${{ secrets.AWS_SDLC_ECR_HOSTNAME }}
-          ecr-image-name: crib-chainlink-untrusted
+          ecr-image-name: ${{ env.ECR_IMAGE_NAME}}
```

Comment:
nit: `space` after the env var :)

Reply:
Addressed in 3d772e6f4ddb0d8ab21f78c7bd1de68bafd7e751